docs(publication): software metapaper (JORS)

wd15 opened the pull request:
Address #441

See [the current version of the paper](https://github.com/wd15/pfhub/blob/issue441-joss/_publications/jors/paper.pdf) to review.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/usnistgov/pfhub/938)
<!-- Reviewable:end -->

tkphd commented:
[JOSS][joss] is hostile toward Federal works, as it does not consider dedication to the public domain to be equivalent to a permissive open-source license. I've renamed the PR to generalize the publication venue, which is more likely to be [JORS][jors].

<!-- References -->
[joss]: http://joss.theoj.org/
[jors]: https://openresearchsoftware.metajnl.com/

tkphd commented:
@wd15, suggested changes are proposed in [your repo](https://github.com/wd15/pfhub/pull/13).

wd15 commented:
@tkpkd: good news, it seem that JORS does not require 3000 words. I misread the criteria. There are two types of papers, one is a "Software Metapaper" and the other is an "Issues in Research Software". It's the latter that requires a full length paper.

tkphd commented:
That's great news!

tkphd commented on _publications/jors/paper.tex:
```diff
@@ -98,91 +96,171 @@ \section*{Introduction}
+    CSS              & 4       \\
+    \hline
+  \end{tabular}
+  \caption{PFHub programming languages and corresponding supported
```
Captions precede tables.

tkphd commented on _publications/jors/paper.tex:
```diff
@@ -98,91 +96,171 @@ \section*{Introduction}
-\textcolor{blue}{E.g. libraries, frameworks, incl. minimum version compatibility.}
+The entire environment can be built using Nix so the only required
+dependency is a functional Nix installation.
 
```
This is only true because Nix pulls in the dependencies. It is also possible to build PFHub *outside* of Nix -- in which case, what dependencies must be satisfied? `yamllint, htmlproofer, jekyll, ...`

tkphd commented on _publications/jors/paper.tex:
```diff
@@ -98,91 +96,171 @@ \section*{Introduction}
-	\item[Date published:] \textcolor{blue}{dd/mm/yy}
+	\item[Name:] Zenodo
+	\item[Persistent identifier:] 10.5281/zenodo.2592705
+	\item[Licence:] NIST Public Domain License~\cite{nistlicense}
```
NIST Software License

tkphd commented on _publications/jors/paper.tex:
```diff
@@ -98,91 +96,171 @@ \section*{Introduction}
-	\item[Licence:] \textcolor{blue}{Open license under which the software is licensed.}
-	\item[Date published:] \textcolor{blue}{dd/mm/yy}
+	\item[Name:] GitHub
+	\item[Persistent identifier:] 10.5281/zenodo.2592705
```
In the example paper, I believe this is the GitHub URL, since the Zenodo DOI won't take you there.

tkphd commented on _publications/jors/paper.tex:
```diff
@@ -98,91 +96,171 @@ \section*{Introduction}
-	\item[Date published:] \textcolor{blue}{dd/mm/yy}
+	\item[Name:] GitHub
+	\item[Persistent identifier:] 10.5281/zenodo.2592705
+	\item[Licence:] NIST Public Domain License~\cite{nistlicense}
```
NIST Software License

wd15 commented:
@tkphd, I hadn't actually finished editing, but I'll try and incorporate those changes. I'll reassign you when I'm finished.

wd15 commented:
@tkphd, over to you. The "reuse" section is a bit weak, but it's looking better.

I'll try and do the dependency graph and see if we can make it look nice enough to include and convey useful information.

wd15 commented:
@drjamesawarren: we're writing a paper about PFHub and we need some text for the funding statement. Is there a standard text (separate from the acknowledgements) recognizing MGI and CHiMaD?

drjamesawarren commented:
If it’s just NIST stuff you don’t have to.  If you want to recognize MGI funding, Albert Davydov has some default verbiage he uses.

Jim

Dr. James A Warren
Director, Materials Genome Program
Material Measurement Laboratory
National Institute of Standards and Technology
(301)-975-5708<tel:(301)-975-5708>

A model is a lie that helps you see the truth. - Howard Skipper

________________________________
On: 14 March 2019 14:02,
"Daniel Wheeler" <notifications@github.com<mailto:notifications@github.com>> wrote:


@drjamesawarren<https://github.com/drjamesawarren>: we're writing a paper about PFHub and we need some text for the funding statement. Is there a standard text (separate from the acknowledgements) recognizing MGI and CHiMaD?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/usnistgov/pfhub/pull/938#issuecomment-472992053>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFF3JD8FJqc-QMYUQW1DSNGPwQ1BCRVzks5vWo6vgaJpZM4bfZpI>.

wd15 commented:
@tkphd: To view the dependency graph use `nix-store -q --graph $(nix-store --realise /nix/store/921clijjp7ai7h87z3wcgkxg2f7rgr0m-pfhub.drv) | dot -T svg -o out.svg`. It's absolutely massive though for PFHub. I have no idea how to reduce the depth short of editing the intermediate dot file by hand. The `dot` command itself doesn't seem to have a method for reducing depth. Also [this doesn't work](https://stackoverflow.com/questions/24733029/create-graph-with-graphviz-limiting-the-depth-of-the-representation).

wd15 commented:
Link to the PDF for review by authors

[paper.pdf](https://github.com/usnistgov/pfhub/files/2988559/paper.pdf)

wd15 commented:
@faical-yannick-congo is the division reader

wd15 commented:
@stvdwtt, thanks for the review. Pretty much everything suggested is included.

I also added the PDF to the repository, [view latest here](https://github.com/wd15/pfhub/blob/issue441-joss/_publications/jors/paper.pdf).

wd15 commented:
Olle approved via email.

wd15 commented:
@dschwen approved [via Gitter](https://gitter.im/usnistgov/chimad-phase-field?at=5c928dbe49fdaa0d81e94aa2)

wd15 commented:
@faical-yannick-congo approved in the NIKE system

guyer commented:
___
*[_publications/jors/paper.tex, line 93 at r10](https://reviewable.io/reviews/usnistgov/pfhub/938#-LaX--ntAVrw4Ul6UePL:-LaX--ntAVrw4Ul6UePM:b16gbyi) ([raw file](https://github.com/usnistgov/pfhub/blob/3cc95cdd10a2a52fc4011c79fe4909b31bc26682/_publications/jors/paper.tex#L93)):*
<details><summary><i alt="summary">Quoted 7 lines of code…</i></summary>

> ```
> $\mu$MAG~\cite{mumag}~\footnote{Certain commercial equipment,
>   instruments, or materials (or suppliers, or software, ...) are
>   identified in this paper to foster understanding. Such
>   identification does not imply recommendation or endorsement by the
>   National Institute of Standards and Technology, nor does it imply
>   that the materials or equipment identified are necessarily the best
>   available for the purpose.\label{disclaimer}}. The reusable PFHub
> ```
</details>


Neither mumag, nor any of the phase field codes footnoted are commercial. No disclaimer required.

On the other hand, figshare, GitHub, and Plotly are commercial. Move footnote to first of these. Not necessary to footnote each one.


<!-- Sent from Reviewable.io -->

tonkmr commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+cross-collaboration between phase-field code developers and
+practitioners by providing a standardized set of benchmark
+problems~\cite{bm1, bm2} along with a web framework for uploading and
+comparing benchmark results from different codes.
```
The introduction does not do a good job of introducing what the paper is actually about. The content of the paper has nothing to do with the phase field method. Rather, it is focused on summarizing the details of the architecture used to facilitate the sharing of results. This focus is clear in the abstract, but this introduction talks about why PFHub is needed by the phase field community, not why a set of tools like this would be useful for any computational community that wants a portal for sharing results. The intro needs to be rewritten to better reflect the focus of the paper.

tonkmr commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+cross-collaboration between phase-field code developers and
+practitioners by providing a standardized set of benchmark
+problems~\cite{bm1, bm2} along with a web framework for uploading and
+comparing benchmark results from different codes.
```
We need a short paragraph clearly stating the purpose of the paper and summarizing what is going to be in it.

wd15 replied:
Thanks for the feedback. You're right, the introduction didn't really make sense in the context of the paper and abstract. I rewrote it, see what you think.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+framework supports phase-field practitioners and code developers
+participating in an effort to improve quality assurance for
+phase-field codes.
+
```
I find this abstract to be too condensed.  I would suggest a rewrite along the lines...

Scientific communities have always struggled with the challenge of efficiently and effectively sharing information.  An online portal provides a valuable space for scientific communities to summarize a shared challenge (e.g., <something>), collect attempts at a solution, and present a quantitative comparison of past attempts in a compelling way. An exemplar of such a portal is $\mu$MAG~\cite{mumag}, <muMag History Blurb>. In this work, we present a description of a generic online hub framework that may be reused and adopted by different scientific communities.  The PFHub framework leverages existing online services to build a static portal website that is considerably easier to deploy and maintain without sacrificing content or scope. The first deployment of the PFHub framework supports phase-field practitioners and code developers participating in an effort to improve quality assurance for phase-field codes. <Value-added statement, like "This deployment has enabled direct comparison of different phase field solver performance and aided in teaching of learners to phase field methodology.">

wd15 replied:
Thanks for the feedback.

Regarding the bare-bones nature of the paper. It is a software meta-paper, which is supposed to be quite short. For example, JOSS asks for between 250 to 1000 words for its meta-papers. JORS doesn't seem to specify anything, but it has another longer paper style which asks for between 3000 and 4000 words so certainly shorter than that. Looking through the software meta-paper articles in the journal, it seem that they are about 1000 to 2000 words. With the latest changes, we are close to 1800 words which is getting quite long for this style of paper (using `detex paper.tex | wc`).

Regarding the abstract, it is actually supposed to be approximately 100 words (from the JORS TeX template). The one you have above is 130 words, which might be too long so I've knocked  it back to 109 words. Anyway, see what you think. It is very similar to what you have above.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+
+\section*{Implementation and architecture}
+
+The PFHub website provides a facility for uploading, displaying and
```
Are we focusing specifically on the PFHub website or more generally on this architecture that can be reused?

wd15 replied:
I've changed the above to PFHub framework and then in the next sentence refer to "This current deployment". I think we're now being consistent about whether we're referring to the framework or the current deployment

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+
+\begin{itemize}
+  \item The users are first required to upload simulation outputs to
+    an archival resource (\emph{e.g.}, Figshare~\footnote{Certain
```
I am thinking all of these different softwares (broadly speaking) need references: Jekyll, Figshare, etc...

wd15 replied:
Both have citations now.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+code and metadata with distributed data records on third-party
+archives avoids the complexity and administrative overhead of
+maintaining a live database and associated back-end application. The
+PFHub infrastructure provides a template for other small scientific
```
"The PFHub infrastructure provides a template for other small scientific communities to host custom content and integrate data from members of their community."

This is the thesis statement.  It should go at the front of the section.

Then, I would state that the general applicability of the PFHub framework will be described in the specific instance of the phase field benchmarking effort.  As such, other communities would likely want to change the exact details of what is included (for example, they might not want benchmark problems in Jupyter notebooks but rather something else) but the general approach and even specific code framework is transferable.

wd15 replied:
I've rewritten the start of the Implementation and Architecture section and have the "The PFHub infrastructure provides ..." sentence at the beginning. Also an extra sentence about how the framework is transferable.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+
+\section*{Quality control}
+
+The framework has a fully automated test recipe deployed on Travis CI
```
Need a topic sentence for this paragraph.  What is QC, why is it important, why is it important to be automated?

wd15 replied:
I'm using the supplied JORS TeX template and the sections are mandatory so "Quality Control" is understood in the context of the journal. I've added a sentence about why an automated test suite / environment is a good idea.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+\vspace{0.5cm}
+\section*{Operating system}
+
+The PFHub framework can be deployed on any platform supporting Nix,
```
Again, topic sentence.

Also, what would someone who is running Windows do?  Suggest a solution.

wd15 replied:
I mentioned that the deployment is platform independent since we don't have servers so not an issue for deploying. Developers would need to use the Docker container on Windows, which is now in the text.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+
+\section*{Software location:}
+
+{\bf Archive}
```
What is this an Archive of?  Data?

wd15 replied:
Again, this is their crazy TeX template. We have to use their section headings. Parts of this are basically filling in a form.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+
+\section*{(3) Reuse potential}
+
+The website infrastructure can be cloned as a Git repository or
```
Topic (introductory) sentence...

wd15 replied:
Again, the topic is understood in the journal, but I've added an intro sentence saying that PFHub can be readily adopted etc.

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+aspects of deploying the framework for a new community.
+
+\begin{itemize}
+  \item Upload new benchmark definitions in a format that Jekyll can
```
Is it really limited to benchmark problems?  If that's the case, that needs to be made clear in the abstract and introduction.

wd15 replied:
Changed to "new data upload specifications"

amjokisaari commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,362 @@
+
+Currently, a deployment for a new community has not been attempted
+and, thus, the above steps need to be refined and documented.
+
```
Perhaps it's not a good idea, but I'm wondering if we should add some kind of section that discusses the success of the website.  Usage statistics, uploads, screenshots of the website...  Some kind of walk-through example so that the software framework described actually has a concrete example to the reader.

wd15 replied:
I've now mentioned that we have 108 separate benchmark uploads in the Implementation and Architecture section. Regarding the statistics, GitHub records a lot of that about the code, downloads and forks etc. I think it's clear to most readers that they can go to GitHub for that sort of meta-data.

I'm not sure what to do for the walk-through example. We already have the workflow outline in the second paragraph of the Implementation section. Also the paper is getting quite long for its purpose. Reviewers / readers can always go to the website and submit model data. I'll point that out in the cover letter with the paper. I'll await your thoughts on this before proceeding.

amjokisaari replied:
I understand from your comments more about the publishing requirements for the intended journal.  As such, I think your changes are good and suggestions for the cover letter.  I wouldn't add anything more to the paper.

tkphd commented:
Excellent feedback, @tonkmr and @amjokisaari. Thanks!

wd15 commented:
Peter approved this via email

wd15 commented:
@drjamesawarren approved via [gitter](https://gitter.im/usnistgov/chimad-phase-field?at=5c92789d2f6ded0abb15a037).

wd15 commented:
@amjokisaari can you review this again after the latest updates. Thanks.

tkphd commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,385 @@
+Office of Basic Energy Sciences, Division of Materials Sciences and
+Engineering under Award \#DE-SC0008637 as part of the Center for
+PRedictive Integrated Structural Materials Science (PRISMS Center) at
+University of Michigan. PWV is grateful for the financial assistance
```
P.W.V.

tkphd commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,385 @@
+PFHub: The Phase-Field Community Hub\\[\baselineskip]
+
+\section*{Paper Authors}
+
```
(Re-label in the Contributors section as well)

tkphd commented:
Nice work, @wd15!

wd15 commented:
ERB approved this, so ready to submit now.

wd15 commented:
[Submitted to JORS](https://openresearchsoftware.metajnl.com/jms/author/submission/276).

wd15 commented:
Finally, we have a review, see [review.txt](https://github.com/usnistgov/pfhub/files/3468089/review.txt). I'll try and make changes in the next few days and have all you review again.

drjamesawarren commented:
Looks good.

Jim

Dr. James A Warren
Director, Materials Genome Program
Material Measurement Laboratory
National Institute of Standards and Technology
(301)-975-5708<tel:(301)-975-5708>

A model is a lie that helps you see the truth. - Howard Skipper

________________________________
On: 05 August 2019 10:43,
"Daniel Wheeler" <notifications@github.com<mailto:notifications@github.com>> wrote:


Finally, we have a review, see review.txt<https://github.com/usnistgov/pfhub/files/3468089/review.txt>. I'll try and make changes in the next few days and have all you review again.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/usnistgov/pfhub/pull/938?email_source=notifications&email_token=ABIXOJCNYPFHQHPLCROUKF3QDA4B7A5CNFSM4G35TJEKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3SBHKA#issuecomment-518263720>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABIXOJBHUEEF2FNFAVGCGPDQDA4B7ANCNFSM4G35TJEA>.

tkphd commented:
The reviewer comments seem reasonable.
Interesting that there is no section for Reviewer B's comments. Is this a superstitious thing, like architects skipping 13 when numbering floors?

guyer commented:
Review B is the arithmetic mean of Review A and Review C

________________________________________
From: Trevor Keller <notifications@github.com>
Sent: Monday, August 5, 2019 1:35 PM
To: usnistgov/pfhub
Cc: Guyer, Jonathan E. Dr. (Fed); Mention
Subject: Re: [usnistgov/pfhub] [WIP] software metapaper (#938)

The reviewer comments seem reasonable.
Interesting that there is no section for Reviewer B's comments. Is this a superstitious thing, like architects skipping 13 when numbering floors?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/usnistgov/pfhub/pull/938?email_source=notifications&email_token=AAN66ROYVFYGYRNSUPYHSDDQDBQEVA5CNFSM4G35TJEKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3SQ2GQ#issuecomment-518327578>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAN66RNGV5ZAWIRIVEK7RWDQDBQEVANCNFSM4G35TJEA>.

wd15 commented:
I've made [these changes](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97) based on the [reviewers' comments](https://github.com/usnistgov/pfhub/files/3468111/review.txt). Point by point,

> 1, Improve the introduction to include better information on a) phase-field codes, b) the phase-field code community, c) the need for a phase-field hub framework, and d) the need for a template for other communities.

Added [paragraph of text](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR120) about phase field which should cover a), b) and c). Added a [paragraph of text](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR143) on why a template for other communities is required.

> 2, Provide more detail on CoffeeScript and on what "appropriate tools" for linting and testing the CoffeeScript front-end are.

Provided [links](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR299) to appropriate tools and [sentence](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR217) about Coffeescript.

> 3, Improve documentation on deployment to include but not limited to screenshots showing the current deployment, and the dependency graph of the project.

Added a bunch of [screenshots](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR260) and [link](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR428) to the development guide. Added a [link that works](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-896b71c74631acb849ed8281e3fd6809R170) to the dependency graph and made it easier to view.

> 4, Make sure that the link to the license is working.

Works for me

> 5, Provide simple input and output data. This does not have to be include in the git repository as it is not useful to have large data files there but could be put on Zenodo.

Added links to [typical upload data](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR204) and [typical display data](https://github.com/usnistgov/pfhub/pull/938/commits/7a27f2ce31222486b9b7549019a34a34d6f86f97#diff-f799a398f3d534fc990396f5c48c34ffR224).

Please review the changes. I'll upload a new version at the paper on Monday next week.

tkphd commented on _publications/joss/paper.md:
```diff
@@ -0,0 +1,162 @@
+
+## Acknowledgments
+
+Acknowledge MGI funding???
```
Now's probably a good time to resolve this question.

wd15 replied:
We're not submitting the JOSS paper only the JORS paper.

guyer commented:
Nitpicky, but current branding calls for "macOS" vs "Mac OS X"

laagesen commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,457 @@
+4. Jokisaari, Andrea M.; C\\
+5. Schwen, Daniel; C\\
+6. Guyer, Jonathan E.; A\\
+7. Aagesen, Larry; C\\
```
Please add middle initial "K"

wd15 replied:
Sorry about that. Should be fixed now.

laagesen commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,457 @@
+available. The PFHub framework provides a template for other
+scientific projects beyond the phase field community. The method
+outlined in this paper of using static infrastructure coupled with
+small independent 3rd party web services providing a flexible approach
```
I'd suggest "provides" instead of "providing" here

wd15 replied:
Fixed

laagesen commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,457 @@
+their community. The current deployment provides a facility for
+uploading, displaying and comparing results from benchmark problems
+supporting phase-field code developers and
+practitioners~\ref{fig:pfhub_frontpage}. However, the framework and
```
Right now in the PDF the reference to the figure gets rendered as "and practitioners 1", so it's not clear the reader is being pointed to look at Figure 1- so unless there is something in the journal's formatting that will capture this, I'd suggest adding `see Figure ref{fig:pfhub_frontpage}`. Same with the references to Figure 3 and 4 below (Figure 2 looks OK)

wd15 replied:
My mistake, thanks for noticing

laagesen commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,457 @@
+  \item The metadata summarizing each simulation is entered into a
+    form on the website, including relevant details such as memory
+    usage, run time and links to the data archived in the first
+    step~\ref{fig:pfhub_upload}.
```
see above comment about clarifying this is supposed to point to a figure

wd15 replied:
done

laagesen commented on _publications/jors/paper.tex:
```diff
@@ -0,0 +1,457 @@
+understand. The interactive plots are displayed using the Plotly
+JavaScript Graphing Library~\cite{plotly} as it provides a
+programmable interface and requires minimal
+configuration~\ref{fig:pfhub_results}, see~\cite{display_data} for
```
see above comment about clarifying this is supposed to point to a figure

wd15 replied:
done

wd15 commented:
I 'vejust submitted a new version. Thanks for all your feedback.

wd15 commented:
I asked for reviews of the proof, but don't feel obliged to do so.

tkphd commented:
Citations in the abstract belong outside the parenthetical abbreviation notes.

wd15 commented:
Proof changes:

 - p5, in list of contributors, names have a pointless capital letter before github IDs
 - p6, ref 24, workows -> workflows
 - p6, merge ref 20 and 23 and they are the same
 - p6, ref 24, add DOI or URL, http://ebooks.iospress.nl/publication/42900

tkphd commented:
Inconsistent use of "phase-field" *vs.* "phase field" throughout

tkphd commented:
Providing both URLs and DOIs on references is redundant.

tkphd commented:
My only objections are very minor.
[proof_TK_mod.pdf](https://github.com/usnistgov/pfhub/files/3610847/proof_TK_mod.pdf)

wd15 commented:
> Inconsistent use of "phase-field" _vs._ "phase field" throughout

I think we decided to go with "phase-field". Is that ok? I found 5 instances of "phase field" to change.

tkphd commented:
Yes, "phase-field" is consistent with the title, and the standard usage.

> I found 5 instances of "phase field"

Gah, I missed one! :sob:

amjokisaari commented:
Congratulations!

On Tue, Sep 24, 2019, 8:49 AM Daniel Wheeler <notifications@github.com>
wrote:

> All done!
>
> https://doi.org/10.5334/jors.276
>
> Thanks for all your help!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/usnistgov/pfhub/pull/938?email_source=notifications&email_token=ABS57FCM7H2JZBCT47WHP3TQLILF3A5CNFSM4G35TJEKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7ONQMI#issuecomment-534566961>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABS57FD562WTBP56Q4VWGNTQLILF3ANCNFSM4G35TJEA>
> .
>

wd15 commented:
Tests only failed here because commit messages are not formatted correctly, but the commit messages were written before the commit message testing went live so I'm merging this. This shouldn't fail on master.
